include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-rcar-common.h

```diff
@@ -82,6 +82,8 @@
 #define IP2SR7(shift, func) IPnSR(2, 7, shift, func)
 #define IP3SR7(shift, func) IPnSR(3, 7, shift, func)
 
+#define IPSR_DUMMY IPnSR(0x1f, 7, 0x1f, 0xf)
```

Comment:
Please add comment similar to `PIN_NOGPSR_START` /  `RCAR_NOGP_PIN`

Reply:
done